config.yml

```diff
@@ -134,6 +134,19 @@
 #
 ##  Retry interval between failed bulk attempts
 #elasticsearch.bulk.retry_interval: 10
+#
+#
+##  Minimal interval of time between MemQueue checks for being full
+#elasticsearch.bulk.mem_queue.refresh_interval: 1
```

Comment:
Why is `mem_queue` under `elasticsearch.bulk`? I'd have expected it under `service`. I thought that everything in `elasticsearch.bulk` is being passed directly to the ES client?